package.json

```diff
@@ -25,7 +25,7 @@
     "test": "jest",
     "typescript": "tsc --noEmit",
     "lint": "eslint \"**/*.{js,ts,tsx}\"",
-    "prepare": "bob build && cp -r src/external/aria lib/external",
```

Comment:
Is there a reason we are removing the `cp` command?

Reply:
you just saved my life,